naughty_strings_test.go

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package ionhash
+
+import (
+	"bufio"
+	"encoding/base64"
+	"os"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/amzn/ion-go/ion"
+)
+
+func TestNaughtyStrings(t *testing.T) {
+	t.Skip() // Skipping test until ion text reader SymbolTable() is implemented
+
+	file, err := os.Open("ion-hash-test/big_list_of_naughty_strings.txt")
```

Comment:
Was the txt file also supposed to be in this PR? 
This txt file is a good candidate for a `test` or `_test_data`  folder that can hold data.

> Note that Go will also ignore directories or files that begin with "." or "_", so you have more flexibility in terms of how you name your test data directory.

Reply:
The txt file lives in the ion-hash-test folder which is a git submodule we added in https://github.com/amzn/ion-hash-go/pull/22/.